draft-ietf-quic-recovery.md

```diff
@@ -330,6 +330,31 @@ min_rtt is the minimum RTT measured over the connection, prior to adjusting by
 ack delay.  Ignoring ack delay for min RTT prevents intentional or unintentional
 underestimation of min RTT, which in turn prevents underestimating smoothed RTT.
 
+A sender calculates both smoothed RTT (SRTT) and RTT variance (RTTVAR) similar
+to those specified in {{?RFC6298}}, see {{on-ack-received}}.
+
+On every newly acknowledged ack-eliciting largest acked:
+~~~
+latest_rtt = ack_time - send_time_of_largest_acked
+~~~
+
+First RTT sample:
+~~~
+min_rtt = latest_rtt
+smoothed_rtt = latest_rtt
```

Comment:
So no ack delay here at all?  That probably needs explanation.

Reply:
For the first sample, using ack_delay will reduce the smoothed_rtt to below min_rtt. I think adding a sentence up in the para above might make sense. How about "Note that smoothed_rtt computation on the first RTT sample does not use ack_delay, because doing so will lead to a smoothed_rtt that is smaller than the min_rtt."